setup.py

```diff
@@ -29,7 +29,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "astropy~=4.0",
+        "astropy~=4.0,>=4.0.2",
```

Comment:
why keeping `~=4.0` if 4.0.2 is the minimal version that solves this probem?

Reply:
Because `~=` means any 4.x version. So this means "any 4.x" version larger than `4.0.2`